test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -1173,6 +1173,14 @@ def capture_comms(self, with_comm_logs=False):
             if self.num_daemon_logs > 0 and with_comm_logs:
                 self.__capture_comm_logs()
 
+        # pbs_comm runs out of server_priv, so copy over 'server_priv' as well
+        pbs_server_priv = os.path.join(self.pbs_home, "server_priv")
+        snap_server_priv = os.path.join(self.snapdir, SVR_PRIV_PATH)
+        core_dir = os.path.join(self.snapdir, CORE_SERVER_PATH)
+        exclude_list = ["accounting"]
+        self.__copy_dir_with_core(pbs_server_priv,
+                                  snap_server_priv, core_dir, exclude_list,
+                                  sudo=self.with_sudo)
```

Comment:
Won't we capture the cores twice?  Once for the server and once for the comms?  I'm not sure there is a good way around that since we can specify what to capture, right?